src/components/Select/Select.jsx

```diff
@@ -18,6 +18,12 @@ import Search from '../Search/Search';
 
 /** Select component that opens a popup menu on click and displays items that can be selected */
 export default class Select extends React.Component {
+  static sameItems = (itemsA, itemsB) =>
+    itemsA.length === itemsB.length &&
+    itemsA.every(
+      (el, ix) => el.id === itemsB[ix].id && el.title === itemsB[ix].title
+    );
+
```

Comment:
Looks like the change from the other branch got pulled in as well 😊 . I merged that one to master, so should be all good 🙌

Reply:
aaaah, I had to add it manually because the app was breaking without it! I guess I forgot to remove it 😅good thing we *do* want it but I'll be more careful next time!